pydm/utilities/__init__.py

```diff
@@ -139,8 +139,7 @@ def find_file(fname, base_path=None, mode=None, extra_path=None):
         The file name. Environment variables, ~ and ~user constructs before
         search.
     base_path : str
-        If None, it defaults to the current open display path or Qt Designer
-        path
+        The directory name of a file pathname from a display, if any
```

Comment:
Did I make it even more confusing haha?? 

From what I noticed, the `base_path` when used in `find_file` is used like this:
`base_path = os.path.dirname(parent_display.loaded_file()) `
or this:
`base_path = os.path.dirname(curr_display.loaded_file())` 

Regardless, this seems to be the directory name of a file's pathname (for now)